unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PassportNavigation/PassportNavigationComponentView.cs

```diff
@@ -43,10 +63,31 @@ public class PassportNavigationComponentView : BaseComponentView, IPassportNavig
 
         public override void Start()
         {
-            subSectionSelector.GetSection(ABOUT_SUB_SECTION_INDEX).onSelect.RemoveAllListeners();
-            subSectionSelector.GetSection(COLLECTIBLES_SUB_SECTION_INDEX).onSelect.RemoveAllListeners();
-            subSectionSelector.GetSection(ABOUT_SUB_SECTION_INDEX).onSelect.AddListener((isActive) => aboutPanel.SetActive(isActive));
-            subSectionSelector.GetSection(COLLECTIBLES_SUB_SECTION_INDEX).onSelect.AddListener((isActive) => wearablesPanel.SetActive(isActive));
+            collectiblesToggle.isOn = false;
+            aboutToggle.isOn = true;
+            aboutToggle.onValueChanged.AddListener((isActive) =>
+            {
+                aboutPanel.SetActive(isActive);
+                aboutText.color = Color.white;
+                collectiblesText.color = Color.black;
+                aboutToggleOn.SetActive(true);
+                aboutToggleOff.SetActive(false);
+                collectiblesToggleOn.SetActive(false);
+                collectiblesToggleOff.SetActive(true);
```

Comment:
Suggestion: group all these lines in a method with a descriptive name of what they do.